src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue18158.cs

```diff
@@ -0,0 +1,27 @@
+#if TEST_FAILS_ON_ANDROID && TEST_FAILS_ON_IOS    //FlyoutLayoutBehavior changes are not supported on Android and iOS
```

Comment:
Could you create a new issue and add a link to it in this comment?

Reply:
@jsuarezruiz , This doesn't appear to be an issue, as the documentation specifies that popover is the only supported behavior on mobile. It looks, this is the expected behavior on both Android and iOS.
![image](https://github.com/user-attachments/assets/10447cbd-d916-4535-9977-28b33b5615f2)